packages/google_maps_flutter/lib/src/bitmap.dart

```diff
@@ -58,8 +58,9 @@ class BitmapDescriptor {
     String assetName, {
     AssetBundle bundle,
     String package,
+    bool mipmaps = true,
   }) async {
-    if (configuration.devicePixelRatio != null) {
+    if (!mipmaps && configuration.devicePixelRatio != null) {
```

Comment:
`!mipmaps` would be false by default and we changed the behavior here right? I think you meant to set mipmaps to false by default?

Reply:
I think true by default makes sense. Feel free to change to whatever you like.